release-notes/6.0/preview/6.0.0-preview.4-runtime-deps.json

```diff
@@ -0,0 +1,1035 @@
+              ]
+            }
+          ]
+        },
+        {
+          "rid": "debian.11",
+          "components": [
+            {
+              "name": "Microsoft.NETCore.App",
+              "type": "Framework",
+              "platformDependencies": [
+                {
+                  "name": "libicu67",
+                  "overrides": {
+                    "name": "libicu57",
+                    "dependencyType": "LinuxPackage"
+                  }
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "rid": "fedora",
+      "components": [
+        {
+          "name": "Microsoft.NETCore.App",
+          "type": "Framework",
+          "platformDependencies": [
+            {
+              "name": "glibc",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "krb5-libs",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libicu",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libgcc",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libstdc++",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "lttng-ust",
+              "dependencyType": "LinuxPackage",
+              "usage": "diagnostics"
+            },
+            {
+              "name": "openssl-libs",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "tzdata",
+              "dependencyType": "LinuxPackage",
+              "usage": "localization"
+            },
+            {
+              "name": "zlib",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            }
+          ]
+        },
+        {
+          "name": "System.DirectoryServices.Protocols",
+          "type": "NuGetPackage",
+          "platformDependencies": [
+            {
+              "name": "openldap",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            }
+          ]
+        },
+        {
+          "name": "System.Drawing.Common",
+          "type": "NuGetPackage",
+          "platformDependencies": [
+            {
+              "name": "cups-libs",
+              "dependencyType": "LinuxPackage",
+              "usage": "printing"
+            },
+            {
+              "name": "libgdiplus:6.0.1",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libx11-dev",
+              "dependencyType": "LinuxPackage",
+              "usage": "xwindows"
```

Comment:
Is there some way to test this behaviour/implementation?  The package name might be `libX11` or `libX11-devel` depending on what it's being used for. Also, with Fedora moving mostly to Wayland, I am not sure how well this would work.

Reply:
See https://github.com/dotnet/core/pull/6170#discussion_r615006176.  The test code I've used to exercise the X11 path is the following:

```csharp
var g = Graphics.FromImage(new Bitmap(10, 10));
g.CopyFromScreen(new Point(0, 0), new Point(1, 1), new Size(1, 1));
```